vendor/tests/test_processor.py

```diff
@@ -854,7 +864,7 @@ def test_create_subscription_success(self):
 
         self.processor.authorize_payment()
 
-        # print(self.processor.transaction_message)
+        print(self.processor.transaction_message)
```

Comment:
did you mean to leave in this print statement?

Reply:
going to comment it again thanks :+1: